synthtool/gcp/templates/python_library/.kokoro/test-samples-impl.sh

```diff
@@ -21,7 +21,7 @@ set -eo pipefail
 shopt -s globstar
 
 # Exit early if samples don't exist
-if ! compgen -G  "./samples/**/requirements.txt" > /dev/null; then
+if ! find samples -name 'requirements.txt' | grep -q .; then
```

Comment:
FWIW, `find` is going to emit a message on stderr for the case that the `samples` directory doesn't exist, e.g.:

```bash
$ find nonesuch -name foobar.txt
find: ‘nonesuch’: No such file or directory
```